packages/logger/tests/unit/Logger.test.ts

```diff
@@ -1,25 +1,25 @@
+import type { LambdaInterface } from '@aws-lambda-powertools/commons/types';
 /**
```

Comment:
Could you please put the comment currently in L2 to L6 at the very top of the file? It's needed for Jest grouping, and it got moved by the Biome formatting.

Reply:
Done :)